granary/microformats2.py

```diff
@@ -85,50 +85,61 @@ def get_text(val):
   return val.get('value') if isinstance(val, dict) else val
 
 
-def object_to_json(obj, ctx={}, trim_nulls=True):
+def object_to_json(obj, trim_nulls=True, entry_class='h-entry',
```

Comment:
actually no, i guess we shouldn't, since it's still used for many other non-activities.